modules/virt-creating-vm-quick-start-web.adoc

```diff
@@ -10,12 +10,6 @@ The web console provides Quick Starts with instructional guided tours for creati
 
 Tasks in a Quick Start begin with selecting a Red Hat template. Then, you can add a boot source and import the operating system image. Finally, you can save the custom template and use it to create a virtual machine.
 
-Quick Start tours for creating virtual machines include the following:
```

Comment:
I think you should leave the Quick Start names. For RHEL, you should use the os-base attribute.

For Windows, I would probably remove "10" so that you are not tied to a version.

Remove the "Importing VMware VM.."